### PR TITLE
Add display of lsp client name to codeaction labels popup

### PR DESCRIPTION
**Pull request by unphased**
This provides an impl for #25 

I still would like to add 

- replace parentheses with a new highlight group for it
- configuration option to not display this (under typical usage with a single lsp we never want this info after all)

### CODE REVIEW

**Comment by luckasRanarison** [comment]
For the highlight group, I think `Comment` would be good.

**Comment by unphased** [comment]
> For the highlight group, I think `Comment` would be good.

Done. Looking good for me

image

**Comment by unphased** [comment]
you think this is good to go in or
- should we disable it by default and enable it with a flag
- add a flag which if specified disables this
?

**Comment by luckasRanarison** [comment]
Looks good, I think I'd be nice to have this by default with the ability to disable it in the config

**Comment by unphased** [comment]
Updated

File: lua/clear-action/popup.lua [hunk 1]
```diff
@@ -28,11 +28,23 @@ M.hide_cursor_autocmd = function()
   })
 end
 
+local function client_name(client_id)
+  local client = vim.lsp.get_client_by_id(client_id)
+  if client then
+    return " " .. client.name
+  else
+    return ""
+  end
```
Comment (luckasRanarison) [inline lua/clear-action/popup.lua:37]: You could make this in one line, and add the space only when concatenating with the row

```suggestion
  return vim.lsp.get_client_by_id(client_id) or ""
```

**Review by luckasRanarison — COMMENTED** [review]
Great! Just added some style suggestions but I could apply them myself if you don't want to.

**Comment by unphased** [comment]
Updated. please double check our width calculation logic. I removed a +1 offset i thought i needed and the size of the popup is perfectly tight now.

**Pull request merged**